### DISCUSSION TITLE
Avoid using  green colour for "long" and red for "short"

### DISCUSSION THREAD

**Discussion by klochowicz** [get10101/itchysats#1597] [Ideas]
Every time I'm trying to go short I pause and reconsider my options because of the red colour of the button (which can be understood as "wrong").

I would strongly suggest using something different:

1. Use the same colour
2. Use a different shade of green or blue (neutral)
3. Pick two different colours (green is also used as "profit" in our app). Two different shades of blue could do the trick nicely.

**Comment by thomaseizinger** [comment]
A related issue is that we also use red & green for the profit text.
